### PR TITLE
[RPA-1879] Disables actions which uses credentials for guests

### PR DESCRIPTION
**Pull request by bhszewczyk**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

**Review by star-in-the-jar — APPROVED** [review]
+ version

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.types.ts [hunk 1]
```diff
@@ -15,6 +15,8 @@ export const ADVANCED_ACTION_GROUP_IDS: ACTION_GROUP[] = [
     ACTION_GROUP.SAP,
     ACTION_GROUP.BEEOFFICE,
     ACTION_GROUP.JIRA,
+    ACTION_GROUP.JIRA_CLOUD,
+    ACTION_GROUP.JIRA_SERVER,
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.types.ts:19]: I think just JIRA is no longer necessary

**Review by star-in-the-jar — APPROVED** [review]
+ version

File: runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useInternalActionsGroups.ts [hunk 2]
```diff
@@ -158,7 +159,7 @@ const useInternalActionsGroups = (): Record<
             ),
             items: Object.values(getCloudFileActions())
         },
-        beeoffice: {
+        beeOffice: {
```
Comment (marcin-depcik) [inline runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useInternalActionsGroups.ts:162]: as long as it doesn't break anything, it's ok

**Review by marcin-depcik — APPROVED** [review]

**Pull request merged**